desktop/apps/article/__tests__/routes.test.js

```diff
@@ -248,6 +290,7 @@ describe('Article Routes', () => {
       )
       const renderLayout = sinon.stub()
       RoutesRewireApi.__Rewire__('renderLayout', renderLayout)
+      req.path = '/series/foobar'
       index(req, res, next)
```

Comment:
Maybe there should be a separate test here for video rather than grouping with /series